integrations.json

```diff
@@ -153,6 +153,25 @@
       }
     ]
   },
+  {
+    "name": "AspNetWebForms",
```

Comment:
Just for naming purposes, we should test this integration with ASP.NET MVC as well. I think it works at a low enough level (ASP.NET) that it applies to any web framework built on top of it. If it does, we could call it simply `AspNet`.

Reply:
Yea, it can be certainly, but we'll have to figure out how to differentiate the OperationName ... I'm sure there's a way, but I couldn't come up with one quickly/easily.  i.e. if you look at the actually integration (AspNetWebFormsIntegration), you'll see that it just uses a AspNetWebFormsHttpModule as the IHttpModule to inject, which is a simple implementation of the base AspNetHttpModule.  The only reason that exists right now is to figure out what the OperationName should be (in this case aspnet-web-forms.request).  There'd have to be a way for us to figure out that the integration was actually a web-forms request vs. something else... at least, I think that's the case

Reply:
TL;DR: drop references to Web Forms. Use `AspNetIntegration` and `aspnet.request`.

The way I see it, this isn't really a "ASP.NET Web Forms" integration but an ASP.NET integration (as in, the lower-level web framework that integrates with IIS and other web frameworks are built upon) which instrument Web Forms request, among others. We should use a generic name that doesn't mention Web Forms, like `AspNetIntegration` for the integration class and `aspnet.request` for the operation name, and let specific web frameworks add their particular spans as children.

Integration | Operation name | Instrumented type
---|---|---
ASP.NET (this one) | `aspnet.request` | `System.Web.HttpApplication` (to add a custom `System.Web.IHttpModule`)
ASP.NET MVC | `aspnet-mvc.request` | `System.Web.Mvc.Async.IAsyncActionInvoker`
ASP.NET Web API | `aspnet-web-api.request` (should be `aspnet-webapi.request`) | `System.Web.Http.Controllers.IHttpController`
ASP.NET Web Forms (future) | `aspnet-webforms.request` | probably `System.Web.UI.Page`
WCF (future) | `wcf.request` | TBD
Nancy (future) | `nancy.request` | TBD

Something similar would happen in .NET Core where "ASP.NET Core MVC" and other libraries are implemented as middleware on top of "ASP.NET Core".

Reply:
done, makes sense naturally once it's laid out.  So I guess this really isn't so much WebForms integration as it is basic ASP.NET, which also is correct.

While I was in there, I noticed the currently WebApi integration operation name is:

aspnet-web-api.request

Should that be changed to just:

aspnet-webapi.request

?

Reply:
> Should that be changed to just: aspnet-webapi.request

Yes, please!